examples/api/amp-access.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const multer = require('multer');
+const upload = multer();
+const nunjucks = require('nunjucks');
+const URL = require('url');
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+examples.use(cookieParser());
+
+const AMP_ACCESS_COOKIE = 'ABE_LOGGED_IN';
+const VALID_USERS = {
+  'mark@gmail.com': true,
+  'jane@gmail.com': true,
+};
+const POWER_USERS = {
+  'Jane@gmail.com': true,
+};
+
+examples.all('/amp-access/authorization', handleAuthorization);
+examples.all('/amp-access/login', handleLogin);
+examples.all('/amp-access/logout', handleLogout);
```

Comment:
Do all of these really need to respond to GET and POST requests?

Reply:
I wasn't sure, but I guess only `/authorization` needs to be POST.